File: Basic Programs/Problem_21.js
/*--------------------------------
Problem 21: Discount Price Calculator\

Write a program that calculates the final price after applying a discount percentage on the original price.
--------------------------------*/

const originalPrice = 300;
const discount = 10;

const discountedAmt = originalPrice * (discount / 100);
const finalPrice = originalPrice - discountedAmt;

console.log(
  `Original Price was: ${originalPrice}, Discount applied: ${discount}%`,
);
console.log(`Final Price is: ${finalPrice}`);

/* OUTPUT: 
Original Price was: 300, Discount applied: 10%
Final Price is: 270
*/

File: shivam-chaurasiya/Practice Problems/Level 1/01 Beginner Programs/Problem21.py
# Discount Price Calculator
# Write a program that calculates the final price after applying a discount percentage on the original price(cost price / marked price) ).

# Discount = Marked Price – Selling Price
# Discount (%) = (Discount/MP) × 100


marked_price = float(input('Enter the Marked Price: '))
discount = float(input('Enter the Discount: '))
selling_price = marked_price - (discount/100)*200
print(selling_price)

